models/db/map.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package db
+
+// KeysInt64 returns keys slice for a map with int64 key
```

Comment:
this helper should move itno golbal utils package ... and can use generics asap golang v1.18 is min requirement

Reply:
Done.